docs/api-guide.md

```diff
@@ -1197,6 +1197,90 @@ In the sample code, the transaction is retried three times maximum and sleeps fo
 
 <div class="notice--info">{{ notice--info | markdownify }}</div>
 
+### Group commit for the Coordinator table
+
+The Coordinator table used for Consensus Commit transactions is a vital data store, and it is recommended to use robust storage for it. However, utilizing more robust storage options, such as internally leveraging multi-AZ or multi-region replication, may lead to increased latency when writing records to the storage, resulting in poor throughput performance. ScalarDB provides a group commit feature for the Coordinator table that groups multiple record writes into a single write operation, improving write throughput. However, it may increase latency.
```

Comment:
Good point. Basically group commit tends to increase latency as it adds an additional step to wait multiple requests. On the other hand, it may reduces workload on the database by reducing request, resulting in reducing the latency. So, we can say it may depends on the situation.

Reply:
I see. How about the following then, which would make it a little more generic?

```suggestion
The Coordinator table that is used for Consensus Commit transactions is a vital data store, and using robust storage for it is recommended. However, utilizing more robust storage options, such as internally leveraging multi-AZ or multi-region replication, may lead to increased latency when writing records to the storage, resulting in poor throughput performance. 

ScalarDB provides a group commit feature for the Coordinator table that groups multiple record writes into a single write operation, improving write throughput. In this case, latency may increase or decrease, depending on your environment.
```

Reply:
> depending on your environment

This sounded a bit too broad scope. How about "depending on the underlying database (or storage) and the workload" or something?

Reply:
Thank you for clarifying! I wasn't sure what the scope was, so your suggestion sounds good to me!

```suggestion
The Coordinator table that is used for Consensus Commit transactions is a vital data store, and using robust storage for it is recommended. However, utilizing more robust storage options, such as internally leveraging multi-AZ or multi-region replication, may lead to increased latency when writing records to the storage, resulting in poor throughput performance.

ScalarDB provides a group commit feature for the Coordinator table that groups multiple record writes into a single write operation, improving write throughput. In this case, latency may increase or decrease, depending on the underlying database and the workload.
```

Reply:
@josh-wong Thank you!